js/graph.js

```diff
@@ -122,7 +121,7 @@ Graph.prototype.draw = function()
 	var i;
 	for(i = -size; i <= size; i += step)
 	{
-		vector[counter + size] = new THREE.Vector3(x.toFixed(2), 0, -this.points[counter + size]);  //FIXME: Somehow the plane is upside-down: the positive y-cordinate is negative
+		vector[counter + size] = new THREE.Vector3(x.toFixed(2), this.points[counter + size], 0.05);
```

Comment:
0.05 ensures that the graphed functions are slightly above the graph itself, though a downside of that is that they're rendered *behind* the graph if you observe it from behind.